packages/playground-examples/copy/ko/JavaScript/External APIs/TypeScript with Node.js

```diff
@@ -0,0 +1,52 @@
+//// { order: 3, isJavaScript: true }
+
+// Node.js는 Chrome 브라우저를 지원하는 자바스크립트 엔진인 v8을 기반으로
```

Comment:
```suggestion
// Node.js는 Chrome 브라우저를 지원하는 JavaScript 엔진인 v8을 기반으로
```
[용어집](https://docs.google.com/spreadsheets/d/1cH_SNZs9-oPcKQWifQEYGUEy-610o4MkZ77YC-Xx5RE/edit#gid=0) 에 따라서 "JavaScript" 로 변경해야 할 것 같습니다 :)